extensions/amp-consent/0.1/amp-consent.js

```diff
@@ -315,6 +321,23 @@ export class AmpConsent extends AMP.BaseElement {
       dev().error(TAG, 'No consent state manager');
       return;
     }
+
+    if (action == ACTION_TYPE.ENTER_FULLSCREEN) {
+      this.vsync_.mutate(() => {
+        this.consentUI_[this.currentDisplayInstance_].enterFullscreen();
+      });
+      return;
+    }
+
+    if (action == ACTION_TYPE.EXIT_FULLSCREEN) {
```

Comment:
I am not expecting a `EXIT_FULLSCREEN` action. The user flow is as following

PROMPT -> ENTER_FULLSCREEN -> USER INTERACTION & HIDE PROMPT